manage.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
@zer0kool this should allow you to use `./manage.py runserver` since it tells the shell to use Python 3.